packages/dos-donts-block/tailwind.config.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
     presets: [require('@frontify/fondue/tailwind'), require('../../tailwind.config')],
-    content: ['src/**/*.{ts,tsx}'],
+    content: ['src/**/*.{ts,tsx}', '../shared/src/**/*.{ts,tsx}'],
```

Comment:
oh. Maybe it would make sense to add this to the readme within shared. Wdyt? Could it also be used like the one from fondue?

Reply:
yeah good idea! Added